CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
    strcpy(dest, src); 
  
-   strncpy(dest,src, strlen(dest)); //DevSkim: ignore DS111237,DS154189 
+   strncpy(dest,src, strlen(dest)); 
    
-   strcat(dest, "this is bad");
+   strcat(dest, "this is bad");     //DevSkim: ignore all
 
    strncat(dest, "this is also bad", strlen(dest)); 
 
    memcpy(dest2, dest, sizeof dest);
 
    malloc(5);
-   
+
+   printf("ahoj");
```

Comment:
[Banned C function detected](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS154189.md#DS154189)
These functions are historically error-prone and have been associated with a significant number of vulnerabilities. Most of these functions have safer alternatives, such as replacing 'strcpy' with 'strlcpy' or 'strcpy_s'.